Uyanda.Coffee.Application/Features/BeverageManagement/Requests/PayCommand.cs

```diff
@@ -7,6 +7,6 @@ namespace Uyanda.Coffee.Application.Features.BeverageManagement.Requests
 {
     public class PayCommand
     {
-        public PayModel Pay { get; set; }
+        public PaymentModel Pay { get; set; }
```

Comment:
I would expect the property name to now be _Payment_

Reply:
Renamed property to _Payment_